### PR TITLE
Update to TestFlight 3.0.0 for iOS

### LINKED ISSUE #14: SDK 3.0.0
TestFlight releases SDK 3.0.0

Could you please update your module?

### PR DESCRIPTION
**Pull request by ericboehs**
Just downloaded TestFlight 3.0.0, put the files in place, built it in Xcode and ran `./build.py`.

I test it on iOS 7.0.6. :+1:

Closes https://github.com/viezel/NappTestFlight/issues/14.

### CODE REVIEW

File: ios/manifest [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 2.1
-apiversion: 2
+version: 3.0.0
+apiversion: 3
```
Comment (viezel) [inline ios/manifest]: Thanks for the update. 
Why do you have this changed? If I remember correctly there is no apiversion 2 for module development in appcelerator. 

you need to change it to:

```
version: 3.0
apiversion: 2
```

File: ios/titanium.xcconfig [hunk 2]
```diff
@@ -4,7 +4,7 @@
 // OF YOUR TITANIUM SDK YOU'RE BUILDING FOR
 //
 //
-TITANIUM_SDK_VERSION = 3.2.0.GA
+TITANIUM_SDK_VERSION = 3.2.2.GA
```
Comment (viezel) [inline ios/titanium.xcconfig]: Why this? 
Do we really need to highten the SDK level. The more we do that - the less apps can use the module.

**Comment by ericboehs** [comment]
Updated.

**Comment by ericboehs** [comment]
Doh, I did. I just forgot to commit it.

**Comment by viezel** [comment]
thanks for the PR ;)

**Comment by ericboehs** [comment]
Was a pleasure. :)

**Pull request merged**